tests/unit/customizations/cloudformation/test_artifact_exporter.py

```diff
@@ -551,6 +557,40 @@ def test_export_cloudformation_stack_no_upload_path_not_file(self):
                 stack_resource.export(resource_id, resource_dict, "dir")
                 self.s3_uploader_mock.upload_with_dedup.assert_not_called()
 
+    def test_export_cloudformation_stack_get_params_relative_path(self):
+        test_template_params = {
+            "Parameter1": "./relative/filepath.txt"
+        }
+        stack_resource = CloudFormationStackResource(self.s3_uploader_mock, template_params=test_template_params)
```

Comment:
I know we've already got lots of code in here that breaks this rule, but please stick to < 80 characters per line.